roles/netbox/handlers/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Shouldn't we have handlers here for reloading/restarting netbox if the config changes?